.github/workflows/update-docs-manual.yml

```diff
@@ -0,0 +1,154 @@
+      need_linux_image_rebuild:   ${{ needs.config.outputs.need_linux_image_rebuild == 'true' }}
+      need_windows_vcpkg_rebuild: ${{ needs.config.outputs.need_windows_vcpkg_rebuild == 'true' }}
+      vcpkg_version:              ${{ needs.config.outputs.vcpkg_version }}
+    secrets: inherit
+
+  create-stubs:
+    needs: [ config, prepare-image ]
+    timeout-minutes: 40
+    runs-on: [self-hosted, linux-arm64]
+    container:
+      image: ${{ matrix.docker-image }}
+      options: --user root
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu22 ]
+        config: [ Release ]
+        compiler: [ clang ]
+        include:
+          - os: ubuntu22
+            docker-image: meshlib/meshlib-ubuntu22-arm64:${{ needs.config.outputs.docker_image_tag }}
+            cxx-compiler: /usr/bin/clang++-14
+            c-compiler: /usr/bin/clang-14
+            cxx-standard: 20
+    permissions:
+      id-token: write # This is required for requesting the JWT
+      contents: read  # This is required for actions/checkout
+
+    steps:
+      - name: Work-around possible permission issues
+        shell: bash
+        run: |
+          # NOTE: {GITHUB_WORKSPACE} != {{ github.workspace }}
+          # Related issue: https://github.com/actions/runner/issues/2058
+          if test -d $GITHUB_WORKSPACE && test -n "$(find ${GITHUB_WORKSPACE} -user root)" ; then
+            mv ${GITHUB_WORKSPACE} ${GITHUB_WORKSPACE}_${RANDOM}
+            mkdir ${GITHUB_WORKSPACE}
+          fi
+
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Checkout third-party submodules
+        run: |
+          # have to checkout selective submodules by our own
+          # related issue: https://github.com/actions/checkout/issues/1779
+          export HOME=${RUNNER_TEMP}
+          git config --global --add safe.directory '*'
+          git submodule update --init --depth 1 thirdparty/imgui thirdparty/parallel-hashmap thirdparty/mrbind-pybind11 thirdparty/mrbind
+
+      - name: Install thirdparty libs
+        run: |
+          ln -s /usr/local/lib/meshlib-thirdparty-lib/lib ./lib
+          ln -s /usr/local/lib/meshlib-thirdparty-lib/include ./include
+
+      - name: Install mrbind
+        run: scripts/mrbind/install_mrbind_ubuntu.sh
+
+      - name: Create virtualenv
+        run: |
+          python3 -m venv .venv
+          . .venv/bin/activate
+          echo $PATH >> $GITHUB_PATH
+
+      - name: Setup python requirements
+        run: python3 -m pip install -r ./requirements/python.txt
+
+      - name: Build
+        run: ./scripts/build_source.sh
+        env:
+          MESHLIB_BUILD_RELEASE: ${{ fromJSON('["OFF", "ON"]')[matrix.config == 'Release'] }}
+          MESHLIB_BUILD_DEBUG: ${{ fromJSON('["OFF", "ON"]')[matrix.config == 'Debug'] }}
+          CMAKE_CXX_COMPILER: ${{ matrix.cxx-compiler }}
+          MR_VERSION: ${{ needs.config.outputs.app_version }}
```

Comment:
Is it required for doc/stubs generation? If not, the `config` workflow usage can be omitted.